scripts/migration/migrate_retraction_dates.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Script to migrate retracted registrations so that their date modified is date of retraction."""
+
+import sys
+import logging
+
+from modularodm import Q
+from website.models import Node, NodeLog
+
+from website.app import init_app
+from scripts import utils as script_utils
+
+logger = logging.getLogger(__name__)
+
+
+def do_migration(logs):
+
+    # ... perform the migration using a list of logs ...
+    for log in logs:
+        when = log.date
+        registration = Node.load(log.params['registration'])
```

Comment:
On staging, only 30 of 185 NodeLogs have `registration` in the params, which would definitely break this. Investigating...